### PR TITLE
Mp4 player

### PR DESCRIPTION
**Pull request by jtwray**
#### npm i video-react
 - includes aria roles labels elements and other a11y accommodations
 - has default styling
 - fullscreen mode
 - plays mp4's

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] Test A[ unit test - 3rd party library renders to the screen playing default video]
- [x] Test B[ unit test - video's src can be updated by passing props from parent component]
- [x] Test C[ Snapshot implemented]

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/utils/axiosWithAuth.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import axios from "axios";
-
 export const axiosWithAuth = () => {
   const token = localStorage.getItem("token");
 
   return axios.create({
+    baseURL: `process.env.BASEURL`,
```
Comment (jimmique-parsons) [inline src/utils/axiosWithAuth.js:6]: I've seen a baseURL in axios but I don't recall seeing it with the env property before. Pretty cool.

**Review by jimmique-parsons — APPROVED** [review]
Code looks good and easy to follow. Excited to see it in action!

File: src/utils/axiosWithAuth.js (hunk 1 above)
Comment (jtwray) [reply src/utils/axiosWithAuth.js:6]: Right!? I thought this was useful to set it dynamically based on the environment. 
--Locally, we can set it to be the local host and whatever port the backend is set to spin up on. ---In production, we can set it to the Hosted backend url on Elastic Beanstalk.
🌠  '

**Review by hisnameisjimmy — COMMENTED** [review]
It looks like, overall, this would power a video list, but I'm not seeing an added action/reducer for an individual page, is that right? Like, you couldn't go to `website.com/video/{id}` using the code in here?

**Comment by jtwray** [comment]
This was originally a larger PR with the video player and the single video route
I broke it down into two smaller PRs. The other dependent on this one
will follow up with PR adding navigation to individual movies
-- by manually entering URL and specific movie id with format website.com/video/{id} 
-- by clicking on preview card from movielist.js using match params

**Pull request merged**